### DISCUSSION TITLE
Rename C symbols to V

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [vlang/v#11768] [Ideas]
When working with c interop, it'd be nice to declare a C function directly as a V function. Instead of having to type `C.` a dozen times for a basic but vast API, can be tiresome.

So I'd like to propose this:

```
#include <myapi.h>
module myapi_bindings

[import: "myapi_something"]
fn myapi_something(int, int) int
```

Especially when creating library wrappers, this would be very helpful!

**Comment by larpon** [comment]
There will maybe be implications with other backends - since they'd have no pure V implementation to work with?

**Comment by JalonSolov** [comment]
Yes.  The `C.` prefix is there to specify calling C functions.  There is also `JS.`, and possibly others coming in the future.

It's really not that terrible, and you can always 'hide' it in a V function and call that instead.

**Comment by danieldaeschle** [comment]
Isn‘t there a way to declare a .c.v file?

**Comment by JalonSolov** [comment]
Just create it.  `foo.c.v`

**Comment by danieldaeschle** [comment]
I mean *declare it in a .c.v file.

**Comment by JalonSolov** [comment]
@danieldaeschle Yes, you can declare it in a .c.v file, but you still have to use the same name in the .v file, unless you do a type alias or whatever.

(FYI:  The message I deleted was an exact duplicate of the one that's still here.  I just removed the duplicate.)

**Comment by MarcTCruz** [comment]
I vote for keeping C. prefix, intention is clear, we know where it's coming from, manual and files are less bloated dividing V functions from C functions.

Readability is better then fast writing, and it doesn't even take time to type the C.

> I mean *declare it in a .c.v file.
@danieldaeschle 

Yes, there is.

_I deleted the message before, so it was not a duplicate, but thanks for explaining._
